luaver

```diff
@@ -156,14 +156,18 @@ __luaver_exists()
 __luaver_download()
 {
     local url=$1
+    local filename=${url##*/}
 
     __luaver_print "Downloading from ${url}"
 
     if __luaver_exists "wget"
     then
-        __luaver_exec_command "wget ${url}"
+        __luaver_exec_command "wget -O ${filename} ${url}"
```

Comment:
To prevent accidential renaming:

```
$ wget https://www.lua.org/ftp/lua-5.3.3.tar.gz
--2016-12-20 18:19:59--  https://www.lua.org/ftp/lua-5.3.3.tar.gz
Resolving www.lua.org... 148.251.24.173
Connecting to www.lua.org|148.251.24.173|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 294290 (287K) [application/gzip]
Saving to: ‘lua-5.3.3.tar.gz’

lua-5.3.3.tar.gz    100%[===================>] 287.39K   235KB/s    in 1.2s

2016-12-20 18:20:01 (235 KB/s) - ‘lua-5.3.3.tar.gz’ saved [294290/294290]

$ wget https://www.lua.org/ftp/lua-5.3.3.tar.gz
--2016-12-20 18:20:02--  https://www.lua.org/ftp/lua-5.3.3.tar.gz
Resolving www.lua.org... 148.251.24.173
Connecting to www.lua.org|148.251.24.173|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 294290 (287K) [application/gzip]
Saving to: ‘lua-5.3.3.tar.gz.1’

lua-5.3.3.tar.gz.1  100%[===================>] 287.39K   160KB/s    in 1.8s

2016-12-20 18:20:05 (160 KB/s) - ‘lua-5.3.3.tar.gz.1’ saved [294290/294290]

$ wget -O lua-5.3.3.tar.gz https://www.lua.org/ftp/lua-5.3.3.tar.gz
--2016-12-20 18:20:48--  https://www.lua.org/ftp/lua-5.3.3.tar.gz
Resolving www.lua.org... 148.251.24.173
Connecting to www.lua.org|148.251.24.173|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 294290 (287K) [application/gzip]
Saving to: ‘lua-5.3.3.tar.gz’

lua-5.3.3.tar.gz    100%[===================>] 287.39K   234KB/s    in 1.2s

2016-12-20 18:20:50 (234 KB/s) - ‘lua-5.3.3.tar.gz’ saved [294290/294290]
```